Add interface guidelines for Breadcrumbs component (#915)
primer/brand: apps/docs/content/components/Breadcrumbs/index.mdx

---
title: Breadcrumbs
description: Breadcrumbs display the current page within the site's hierarchy, allowing users to easily navigate to a different level.
---

import ComponentLayout from '../../../src/layouts/component-layout'

import anatomy from './images/anatomy.png'
import variants from './images/variants.png'
import doOverflow from './images/do-overflow.png'
import dontOverflow from './images/dont-overflow.png'

export default ComponentLayout

## Anatomy

<img
  src={anatomy}
  alt="Breadcrumbs with location-based annotations for parent-level links, divider, and current page."
/>

## Usage

Use breadcrumbs to display a user's path within a site's hierarchy, helping them understand their location and easily navigate back to higher-level pages, especially on sites with deep structures.

Breadcrumbs are most appropriate on pages that:

> - Do not have a section-level navigation
> - May need the ability to quickly go back to the previous (parent) page

Reviewer: Could we give an example of section level navigation? Guessing you mean SubNav?

```suggestion
- Are missing section-level navigation elements E.g. [SubNav](/components/SubNav)
- Need to surface contextually-relevant content through its parent page(s)
```

  </Dont>
</DoDontContainer>

## Options

### Variants

> The breadcrumbs component is available in two variants: a default variant, where links use the standard text color, positioning the navigation as a secondary element on the page, and an accent variant, which emphasizes parent links with an accent color. The default variant is suitable for most scenarios, while the accent variant should be reserved for pages where the breadcrumbs are essential for navigation.

Reviewer: Suggestion to break this up for easier reading.

```suggestion
The breadcrumbs component is available in two variants:

- The default variant uses the standard text color, positioning the navigation as a secondary element on the page. This is suitable for most scenarios.  
- The accent variant emphasizes parent links with an accent color. This variant should should be reserved for pages where the breadcrumbs are essential for navigation.
```
